app/src/main/java/uk/ac/ebi/atlas/trader/ScxaExperimentRepository.java

```diff
@@ -44,7 +44,8 @@ public Experiment getExperiment(String experimentAccession) {
                     "Experiment with accession " + experimentAccession + " could not be found");
         }
 
-        if (experimentDto.getExperimentType() != ExperimentType.SINGLE_CELL_RNASEQ_MRNA_BASELINE) {
+        if (experimentDto.getExperimentType() != ExperimentType.SINGLE_CELL_RNASEQ_MRNA_BASELINE &&
+                experimentDto.getExperimentType() != ExperimentType.SINGLE_NUCLEUS_RNASEQ_MRNA_BASELINE) {
```

Comment:
This can (and should) be simplified to:
```java
if (!experimentDto.getExperimentType().isSingleCell())
```

And that will be always fine when we add new experiment types.